app/renderer/components/urlBarIcon.js

```diff
@@ -11,6 +11,19 @@ const dndData = require('../../../js/dndData')
 const {isSourceAboutUrl} = require('../../../js/lib/appUrlUtil')
 const searchIconSize = 16
 
+const getIconCssClass = (ctx) => {
+  if (ctx.isSearch) {
```

Comment:
I am not able to understand the use case here. Can you please explain the issue u r seeing?